extensions/resteasy-reactive/rest-client-reactive/runtime/src/main/java/io/quarkus/rest/client/reactive/runtime/ClientBlockingResponseFilter.java

```diff
@@ -0,0 +1,37 @@
+package io.quarkus.rest.client.reactive.runtime;
+
+import java.io.IOException;
+import java.util.concurrent.Executor;
+import java.util.function.Supplier;
+
+import jakarta.ws.rs.client.ClientRequestContext;
+import jakarta.ws.rs.client.ClientResponseContext;
+import jakarta.ws.rs.client.ClientResponseFilter;
+
+import org.jboss.resteasy.reactive.client.impl.ClientRequestContextImpl;
+import org.jboss.resteasy.reactive.client.impl.RestClientRequestContext;
+
+import io.quarkus.runtime.ExecutorRecorder;
+import io.vertx.core.Context;
+
+public class ClientBlockingResponseFilter implements ClientResponseFilter {
```

Comment:
It would probably make sense to add some Javadoc as well

Reply:
Added